docs/ref/forms/models.txt

```diff
@@ -1,15 +1,258 @@
-====================
-Model Form Functions
-====================
-
-Model Form API reference. For introductory material about model forms, see the
-:doc:`/topics/forms/modelforms` topic guide.
+===========
+ModelForms
+===========
 
 .. module:: django.forms.models
-   :synopsis: Django's functions for building model forms and formsets.
+    :synopsis: ModelForm API reference for inner ``Meta`` class and factory
+        functions
+
+.. class:: ModelForm
+
+.. _ref-modelform:
+
+.. currentmodule:: django.forms
+
+:class:`~django.forms.ModelForm` API reference. For introductory material about
+ModelForms, see the :doc:`/topics/forms/modelforms` topic guide.
+
+.. _ref-modelform-meta-options:
+
+ModelForm ``Meta`` Options
+==========================
+
+The :class:`~django.forms.ModelForm` helper class can be used to build forms
+that closely reflect specific Django models. The exact structure of the form
+produced can be controlled by setting metadata options as attributes of an inner
+``Meta`` class.
+
+::
+
+    # Example using some sort of Book model
+
+    from django.forms import ModelForm
+    from myLibraryApp.models import Book
```

Comment:
Nitpick comment:

```suggestion
    from myapp.models import Book
```
It looks like the majority of code examples would use `myapp`.

Reply:
Good point-- I should be consistent with that.

As a sidebar, is there a convention anywhere in the docs for contributors about what can/should be used as an example (where possible) for consistency across the docs?

Reply:
Ya know I checked the codebase and I didn't see perfect uniformity. However, `myapp` was by far the most common. I probably wouldn't have said something if it were using snake case either.